recipes/mp-units/2.2.0/conanfile.py

```diff
@@ -0,0 +1,266 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import default_cppstd, valid_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+
+required_conan_version = ">=1.59.0"
+
+
+def loose_lt_semver(v1, v2):
+    lv1 = [int(v) for v in v1.split(".")]
+    lv2 = [int(v) for v in v2.split(".")]
+    min_length = min(len(lv1), len(lv2))
+    return lv1[:min_length] < lv2[:min_length]
+
+
+class MPUnitsConan(ConanFile):
+    name = "mp-units"
+    homepage = "https://github.com/mpusz/mp-units"
+    description = "The quantities and units library for C++"
+    topics = (
+        "units",
+        "dimensions",
+        "quantities",
+        "dimensional-analysis",
+        "physical-quantities",
+        "physical-units",
+        "system-of-units",
+        "system-of-quantities",
+        "isq",
+        "si",
+        "library",
+        "quantity-manipulation",
+    )
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "cxx_modules": ["auto", True, False],
+        "std_format": ["auto", True, False],
+        "string_view_ret": ["auto", True, False],
+        "no_crtp": ["auto", True, False],
+        "contracts": ["none", "gsl-lite", "ms-gsl"],
```

Comment:
Is this `none` an option of "Not using contracts", or "Not setting the option so nothing is used"? In the second case, using a proper `None` would be the preferred way

Reply:
It is meant to mean "Not using contracts". But I understand your concern and I will change it to `False`. Especially that I hope to provide `True` when C++26 is released :-)

Reply:
After a longer thought I decided to stay with "none" here. If C++26 will support contracts then I will rename the entire option to `std_contracts` and make it compatible with `std_format`